src/SFA.DAS.ApprenticeFeedback.Data.UnitTests/DatabaseMock/MoqExtensions.cs

```diff
@@ -36,7 +36,7 @@ private static void ConfigureDbSetCalls<TEntity>(this Mock<DbSet<TEntity>> mock)
 			where TEntity : class
 		{
 			mock.Setup(m => m.AsQueryable()).Returns(mock.Object);
-			mock.Setup(m => m.AsAsyncEnumerable()).Returns(mock.Object);
+			mock.Setup(m => m.AsAsyncEnumerable()).Returns(mock.Object.AsAsyncEnumerable());
```

Comment:
In .NET 6, `AsAsyncEnumerable()` now needs to be explicitly called. Otherwise this line will not compile.